docs/command-reference/strings/bitpos.md

```diff
@@ -54,21 +52,19 @@ However, this behavior changes if you are looking for clear bits and specify a r
 dragonfly> SET mykey "\xff\xf0\x00"
 "OK"
 dragonfly> BITPOS mykey 0
-(integer) 0
+(integer) 12 
```

Comment:
this seems wrong

Reply:
@talbii  - was it fixed?

Reply:
It looks correct to me, the string "\xff\xf0\x00" in binary is `1111'1111'1111'0000..`, and the first position in which we have `0` is `12`. I also checked with Redis and it seems to agree.

Reply:
sorry - :)